lib/fennec/auth.ex

```diff
@@ -0,0 +1,77 @@
+defmodule Fennec.Auth do
+  @moduledoc false
+  # This module implements authentication and authorization of
+  # the STUN and TURN protocol.
+
+  alias Jerboa.Format.Body.Attribute
+  alias Jerboa.Format.Body.Attribute.{Username, Nonce, Realm}
+  alias Jerboa.Params
+
+  @nonce_bytes 48
+  @nonce_lifetime_seconds 60 * 60 # 1h
```

Comment:
Is it recommended lifetime?

I know that it's initial implementation, but it would be nice to have those values configurable in the future 🙂